driver-examples/tycho/com.datastax.driver.examples.rcp.mailbox/src/main/java/com/datastax/driver/examples/rcp/mailbox/MailboxActivator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.examples.rcp.mailbox;
+
+import java.util.Hashtable;
+
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.examples.rcp.mailbox.impl.MailboxServiceImpl;
+
+public class MailboxActivator implements BundleActivator {
```

Comment:
Indeed, but I think examples should stay simple and straightforward, so I'd be more in favor of duplicating these classes.